src/dosbox.cpp

```diff
@@ -1169,6 +1169,12 @@ void DOSBOX_Init()
 	pstring->Set_help(
 	        "Language code of the keyboard layout, or 'auto' ('auto' by default).");
 
+	pstring = secprop->Add_path("setver_table_file", only_at_start, "");
```

Comment:
Maybe we should just pick a name for the database file, without giving users option for changing the path?

Reply:
I would prefer to have such option, as I am storing the database inside my virtual drive C: - this makes archiving my guest system slightly easier, and I can check version override using DOS Navigator.